libs/core/src/lib/card/card.component.scss

```diff
@@ -17,6 +17,9 @@
     justify-content: flex-end;
     border-top: 0.0625rem solid #ccc;
     border-top: 0.0625rem solid var(--sapTile_SeparatorColor, #ccc);
+    min-height: 2.5rem;
+    height: 2.5rem;
+    align-items: center;
```

Comment:
Is this css brought in Fundamental-styles?

Reply:
not yet.
will raise a different PR in fundamental-styles to move all classes in this .scss file.